src/replica.rs

```diff
@@ -92,10 +94,8 @@ impl Replica {
     }
 
     #[cfg(test)]
-    pub fn new_inmemory() -> Replica {
-        use crate::StorageConfig;
-
-        Replica::new(StorageConfig::InMemory.into_storage().unwrap())
+    pub(crate) fn new_inmemory() -> Replica<InMemoryStorage> {
```

Comment:
Let's make this an easy followup PR: this function isn't necessary and is only used in tests anyway, so its uses can be replaced by its body.